### PR TITLE
Release blog 4.7

### PR DESCRIPTION
**Pull request by MinThaMie**
Release blogpost 4.7

### CODE REVIEW

File: content/ember-released-4-7.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+date: 2022-09-5T00:00:00.000Z
+tags:
+  - releases
+  - '2022'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.7 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 4.7 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
Comment (runspired) [inline content/ember-released-4-7.md]:
```suggestion
This release kicks off the 4.8 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

File: content/ember-released-4-7.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+- [#8093](https://github.com/emberjs/data/pull/8093) Implement helper deprecations ([RFC 742](https://rfcs.emberjs.com/id/0742-ember-data-deprecate-helper-functions)).
+- [#8092](https://github.com/emberjs/data/pull/8092) Deprecate `Model.reopen`/`reopenClass` and eager static fields lookups (implements [RFC 738](https://rfcs.emberjs.com/id/0738-ember-data-deprecate-model-reopen) and [RFC 741](https://rfcs.emberjs.com/id/0741-ember-data-deprecate-model-static-field-access-without-lookup)).
+- [#8084](https://github.com/emberjs/data/pull/8084) Eliminate InternalModel, nearly all private API's have undergone significant change, if your app previously used these APIs the most likely refactor is to use a custom model or a custom cache.
+- [#8115](https://github.com/emberjs/data/pull/8115) Implement strict relationships ([RFC 739](https://rfcs.emberjs.com/id/0739-ember-data-deprecate-non-strict-relationships)).
+
+Support for the `toJSON` method on Ember Data records has been removed. It was deprecated in `3.x` and slated for removal in `4.x`.
```
Comment (runspired) [inline content/ember-released-4-7.md]: this occurred a while back, and support was removed in 4.0, not sure how it is getting into this blog post

File: content/ember-released-4-7.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+#### Features
+
+Ember Data 4.7 introduced 2 features.
+
+- [#7955](https://github.com/emberjs/data/pull/7955) Explicit Polymorphic Relationship Support [emberjs/rfcs#793](https://rfcs.emberjs.com/id/0793-polymporphic-relations-without-inheritance)
+- [#8134](https://github.com/emberjs/data/pull/8134) The return values of hasMany relationships, peekAll, findAll and query are now proxies to native arrays and as such all native array APIs are now usable. These objects will act fully as if they are native arrays. Restrictions on immutability of the result of peekAll and query still apply ([RFC 846](https://rfcs.emberjs.com/id/0846-ember-data-deprecate-proxies)).
+
+#### Deprecations
+
+Ember Data 4.7 introduced 8 deprecations. Additional details are in the [release notes](https://github.com/emberjs/data/releases/tag/v4.7.1).
```
Comment (runspired) [inline content/ember-released-4-7.md]: while there are only 8 PRs implementing deprecations there are 13 total. They are:

- deprecate model.reopen [emberjs/rfcs#738](https://github.com/emberjs/rfcs/pull/738)
- deprecate early static [emberjs/rfcs#741](https://github.com/emberjs/rfcs/pull/741)
- deprecate importable helper functions [emberjs/rfcs#742](https://github.com/emberjs/rfcs/pull/742)
- deprecate non-promise promise-many-array APIs  [emberjs/rfcs#745](https://github.com/emberjs/rfcs/pull/745)
- deprecate v1-cache store APIs  [emberjs/rfcs#461](https://github.com/emberjs/rfcs/pull/461)
- deprecate v1-cache  [emberjs/rfcs#461](https://github.com/emberjs/rfcs/pull/461)
- deprecate using `A()` on RecordArray/ManyArray/PromiseManyArray  [emberjs/rfcs#846](https://github.com/emberjs/rfcs/pull/846)
- deprecate relationships without a specified type [emberjs/rfcs#739](https://github.com/emberjs/rfcs/pull/739)
- deprecate relationships without specified async behavior [emberjs/rfcs#739](https://github.com/emberjs/rfcs/pull/739)
- deprecate relationships without an explicit inverse  [emberjs/rfcs#739](https://github.com/emberjs/rfcs/pull/739)
- deprecate polymorphic relationships without explicit polymorphic flags [emberjs/rfcs#793](https://github.com/emberjs/rfcs/pull/793)
- deprecate promise-array and promise-object return values [emberjs/rfcs#846](https://github.com/emberjs/rfcs/pull/846)
- deprecate array-like APIs (non native array APIs) on array proxies [emberjs/rfcs#846](https://github.com/emberjs/rfcs/pull/846)

File: content/ember-released-4-7.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 4.7
+
+Version 4.7 implements a "2.1" cache spec while deprecating the v1 cache spec. This spec differs somewhat from the [original 2.0 RFC](https://rfcs.emberjs.com/id/0461-ember-data-singleton-record-data), an updated RFC specifying the modifications will be published soon. Users looking to migrate to the 2.1 cache should expect that the finalized version of 2.1 enforces that the cache is always a singleton (vs merely allowed to be a singleton), and that data provided to the store and queries to the store from the cache should always contain identifiers in their stable form. This latter point is enforced in 4.7, the former is not yet enforced.
```
Comment (runspired) [inline content/ember-released-4-7.md:59]: The performance changes are the big callout to make, you can probably just directly copy that portion of the release notes:  https://github.com/emberjs/data/releases/tag/v4.7.1

**Review by runspired — APPROVED** [review]
Overall looks good, left some notes

File: content/ember-released-4-7.md (hunk 3 above)
Comment (MinThaMie) [reply content/ember-released-4-7.md]: Cool I'll change it to 13 and people could find the docs in the release notes correct?

File: content/ember-released-4-7.md (hunk 4 above)
Comment (MinThaMie) [reply content/ember-released-4-7.md:59]: Cool, will add that too

File: content/ember-released-4-7.md (hunk 2 above)
Comment (MinThaMie) [reply content/ember-released-4-7.md]: This was automatically generated. I've removed it!

**Review by mansona — APPROVED** [review]
Looks good to me 🎉

**Pull request merged**